test/Regex/characterclass_with_range.js

```diff
@@ -0,0 +1,163 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+WScript.LoadScriptFile("..\\UnitTestFramework\\UnitTestFramework.js");
+
+let re = /^[\s-a-z]$/;

```

Comment:
Is this regex expected to include codepoints between the last of the spaces and `a`? That is, do the numbers and capital letters fall into that range because they are between `0x20 = ' '` and `0x61 = 'a'`?

Reply:
No, sorry I should have linked the issue initially. https://github.com/Microsoft/ChakraCore/issues/258 
/[\s-a]/ should just be interpreted as the set of all whitespace, '-', and the letter a